src/Microsoft.DotNet.Darc/DarcLib/Helpers/StringUtils.cs

```diff
@@ -75,4 +79,72 @@ public static string GetXxHash64(string input)
         byte[] hashBytes = hasher.GetCurrentHash();
         return Convert.ToHexString(hashBytes);
     }
+
+    public static string BuildDependencyUpdateCommitMessage(IEnumerable<DependencyUpdate> updates)
```

Comment:
Don't think this method should be in here, as it feels too specific comparing to what we have?
Would it fit better in the PullRequestBuilder or someplace like that?